testing/ostinato/ModelTest.patch

```diff
@@ -0,0 +1,16 @@
+Undefined reference to ModelTest::ModelTest under defaults.
+Clearly, we're not in a debug mode, but the sources think we are.
+diff -ur a/client/ostinato.pro b/client/ostinato.pro
```

Comment:
Is this upstream?

Reply:
No, the patch is made by SpaceToast

Reply:
Maybe it'd be a good idea mentioning that in the patch header then, makes it easier for other people to touch this :)